src/planner/vectorfield/VectorFieldPlanner.cpp

```diff
@@ -141,6 +141,11 @@ std::unique_ptr<aikido::trajectory::Spline> planToEndEffectorOffset(
     std::chrono::duration<double> timelimit,
     planner::PlanningResult* planningResult)
 {
+
+  // ensure that no two planners run at the same time
+  auto robot = metaskeleton->getBodyNode(0)->getSkeleton();
```

Comment:
It should return `nullptr` (with warning) or throw an exception if the number of bodies is zero.

Reply:
good idea :)
done